test/OrchardCore.Tests/Localization/PortableObjectStringLocalizerFactoryTests.cs

```diff
@@ -13,9 +13,10 @@ namespace OrchardCore.Tests.Localization
 {
     public class PortableObjectStringLocalizerFactoryTests
     {
-        [Fact]
-        public async Task LocalizerReturnsTranslationFromInnerClass()
```

Comment:
Probably related with Date format translation issue.

Reply:
No, this another one, basically to resolve the issue when the translation in inner class